packages/amplication-client/src/Resource/ResourceMenu.tsx

```diff
@@ -37,27 +36,27 @@ const ResourceMenu: React.FC<{}> = () => {
           >
             <CircleBadge
```

Comment:
use resourceCircleBadge and option to either show initials or icon (or just use Icon and let's get rid of the initials). @alexbass86 what do you say?

Reply:
It will be in a separated issue